CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
-### 0.8.2 (Next)
+### 0.9.1 (Next)
```

Comment:
For the prep part we remove the next section so that if you look at the changelog in the released branch it just says 0.9.0 on top.